e107_admin/mailout.php

```diff
@@ -1586,7 +1586,7 @@ function options($parms, $value, $id, $attributes)
 			$preview = e_SELF."?mode=main&action=preview&id=".$id;
 			$text .= "<span class='btn-group'>";
 			$text .= "<a href='".$link."' class='btn btn-default' title='".LAN_MAILOUT_08."'>".E_32_MAIL."</a>";
-			$text .= "<a rel='external' class='btn btn-default e-modal' data-modal-caption='Email preview' href='".$preview."'  title='".LAN_PREVIEW."'>".E_32_SEARCH."</a>";
+			$text .= "<a rel='external' class='btn btn-default e-modal' data-modal-caption='Email ".LAN_PREVIEW."' href='".$preview."'  title='".LAN_PREVIEW."'>".E_32_SEARCH."</a>";
```

Comment:
This can just be changed to LAN_PREVIEW. 
`data-modal-caption='".LAN_PREVIEW."'  `